### ISSUE TITLE
Holdings and Availability Display

### ISSUE THREAD

**Issue by cdmo** [psu-libraries/psulib_blacklight#196]
Holdings and summary holdings

https://github.com/psu-libraries/psulib_blacklight/wiki/Holdings-and-Availability-Display

**Comment by banukutlu** [comment]
Connected to https://github.com/psu-libraries/psulib_traject/pull/61

**Comment by banukutlu** [comment]
@ruthtillman 497977 has a holding with the below ItemInfo:
```
CallInfo>
<libraryID>ALTOONA</libraryID>
<classificationID>LC</classificationID>
<callNumber>BF76.7.P83 2010</callNumber>
<numberOfCopies>6</numberOfCopies>
...
<ItemInfo>
<itemID>000066834199</itemID>
<itemTypeID>BOOKFLOAT</itemTypeID>
<currentLocationID>ILLEND</currentLocationID>
<homeLocationID>STACKS-BR</homeLocationID>
<dueDate>2019-05-14T00:00:00.922-04:00</dueDate>
<chargeable>false</chargeable>
<numberOfHolds>1</numberOfHolds>
<fixedTimeBooking>false</fixedTimeBooking>
</ItemInfo>```

homeLocationID is Beaver - Stacks but currentLocationID is `"ILLEND": "This copy unavailable, submit request via Interlibrary Loan"` and LibraryID is ALTOONA. 

Since we are grouping by LibraryID, this holding shows up under Altoona:

![Screen Shot 2019-05-20 at 11 41 04 AM](https://user-images.githubusercontent.com/6477867/58037596-b98e4700-7afb-11e9-97b0-5f023f055aa4.png)

**Comment by banukutlu** [comment]
@ruthtillman 52888 has a callInfo with `<libraryID>SERIAL-SRV</libraryID>`, SERIAL-SRV is not in the locations list that we were using for bento availability which we currently use while implementing BL availability. We talked about updating the locations list but also other than updating, what should we do in such cases if there is a libraryID that won't be in our list? Is this something we account for or not? Do we not list the holding and not count for available items?

**Comment by ruthtillman** [comment]
I addressed the first part in the wiki. Re: `497977` -- considering.

**Comment by ruthtillman** [comment]
Updated wiki in consultation with Banu.

**Comment by cdmo** [comment]
@ruthtillman Looking at populating the ILL form via [the URL mentioned in the wiki](https://github.com/psu-libraries/psulib_blacklight/wiki/Holdings-and-Availability-Display#displaying-illend), the URL there lists call number, however the form at the endpoint doesn't have a field for call number. It has OCLC number and ISBN, but no call...

I'm wondering if maybe ILL doesn't need this data any more and we can just omit it?

**Comment by ruthtillman** [comment]
@cdmo I think esp for now we can omit it.

**Comment by cdmo** [comment]
Yup was thinking the same, review with ILL folks. We could pass along other field values, but I don't know what's important to their process. Probably ISBN and title/author/pubdate are good, but...

**Comment by banukutlu** [comment]
@ruthtillman could you check `370754` availability? there are two copies available one Book and the other one is Microform, since we do not display the type with availability, it seems like the two copies are same.

![Screen Shot 2019-06-03 at 4 15 54 PM](https://user-images.githubusercontent.com/6477867/58831548-eaea3500-861a-11e9-9f76-efd7ff3a0d5e.png)

**Comment by ruthtillman** [comment]
Updating wiki page with various things and will note here. This is just a styling note:

Please remove the hover styling from items listed in holdings. It gives the impression one should be able to do something by clicking, but one cannot.

**Comment by ruthtillman** [comment]
Updates to wiki page:

Added info about item type. I agree re: comment about `370754`, we need to differentiate. I've added the field + the URL to where we keep the dictionary for the ItemTypeID values. https://github.com/psu-libraries/psulib_blacklight/wiki/Holdings-and-Availability-Display#item-info also updated display section to note we're displaying call number, ItemTypeID (mapped to display text), and availability status.
